x-pack/functionbeat/functionbeat.yml

```diff
@@ -0,0 +1,158 @@
+###################### Functionbeat Configuration Example #######################
+
+# This file is an example configuration file highlighting only the most common
+# options. The functionbeat.reference.yml file from the same directory contains all the
+# supported options with more comments. You can use it as a reference.
+#
+# You can find the full configuration reference here:
+# https://www.elastic.co/guide/en/beats/functionbeat/index.html
+#
+
+#============================  Provider ===============================
+# Configure functions to run on AWS Lambda, currently we assume that the credentials
+# are present in the environment to correctly create the function when using the CLI.
+#
+functionbeat.provider.aws.functions:
+# Accepts events from a cloudwatch log group.
+  - name: fn_cloudwatch_logs
```

Comment:
This default also causes this: `Function: fn_cloudwatch_logs, could not deploy, error: ValidationError: 1 validation error detected: Value 'fnb-fn_cloudwatch_logs-stack' at 'stackName' failed to satisfy constraint: Member must satisfy regular expression pattern: [a-zA-Z][-a-zA-Z0-9]*`

I suggest renaming it to just `cloudwatch`? It also simplifies the deploy call: `functionbeat deploy cloudwatch'

Reply:
I have fixed that in https://github.com/elastic/beats/pull/8713